<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextCodecConvertDialog</class>
 <widget class="QDialog" name="TextCodecConvertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>928</width>
    <height>471</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>转换文本编码</string>
  </property>
  <layout class="QGridLayout" name="gridLayout" columnstretch="0,1,0,0,1,0">
   <item row="1" column="2">
    <widget class="QPushButton" name="availableCodecButton_source">
     <property name="text">
      <string>支持的编码</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QComboBox" name="sourceTextCodecComboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>Shift-JIS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GB18030</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Big5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UTF-8</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="0" column="3" colspan="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>转换后文本预览：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="5">
    <widget class="QPushButton" name="availableCodecButton_target">
     <property name="text">
      <string>支持的编码</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="6">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QTextBrowser" name="sourceTextBrowser"/>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>转换前文本：</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="3">
    <widget class="QTextBrowser" name="targetTextBrowser"/>
   </item>
   <item row="1" column="3" colspan="2">
    <widget class="QComboBox" name="targetTextCodecComboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>Shift-JIS</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>GB18030</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Big5</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>UTF-8</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="3" column="0" colspan="6">
    <widget class="QCheckBox" name="showSystemTextCodecCheckBox">
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;打开此项后，程序将在预览窗口中同时输出系统编码下对您选择编码字符串的重解码后的显示效果。对文件名转换来说，此项十分有帮助。&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>在预览中输出在系统编码下重解码的预览</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TextCodecConvertDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TextCodecConvertDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
